src/api/app/views/webui/user/user_profile_redesign/_basic_info.html.haml

```diff
@@ -3,7 +3,7 @@
            locals: { displayed_user: user, role_titles: role_titles, account_edit_link: account_edit_link }
 
 .basic-info
-  - if configuration.accounts_editable?(user)
+  - if configuration.accounts_editable?(user) && policy(user).update?
```

Comment:
Shouldn't also this check also be added [here](https://github.com/openSUSE/open-build-service/blob/dc04932fa9e9f515c74e5b22d2aecf04a1399264/src/api/app/views/webui/user/_index_actions.html.haml#L2), then?

Reply:
True! I've just checked that, without the feature flag enabled, I can edit my own profile but another user's one too.

Reply:
@danidoni can you please add the same condition to [here](https://github.com/openSUSE/open-build-service/blob/dc04932fa9e9f515c74e5b22d2aecf04a1399264/src/api/app/views/webui/user/_index_actions.html.haml#L2) ?

Reply:
Yes